samples/client/petstore/kotlin/http/client/.openapi-generator-ignore

```diff
@@ -0,0 +1,23 @@
+# OpenAPI Generator Ignore
```

Comment:
Why are these other files changing?

Reply:
Well it's a good question, they shouldn't. Though they did once I ran the commands that are requested above

```
./mvnw clean package 
./bin/generate-samples.sh
./bin/utils/export_docs_generators.sh
```

How do you suggest to complete this PR? 

I'm happy to remove the other files, and keep it to the one line change. Does it something else?

Reply:
Those are indeed remnants from another work I was doing. Closing this to https://github.com/OpenAPITools/openapi-generator/pull/14298